syntax = "proto3";

package helloservice;

// Define your service with RPC methods
service Greeter {
  rpc SayHello (HelloRequest) returns (HelloResponse) {}
  rpc SayHelloStream (HelloRequest) returns (stream HelloResponse) {}
}

// Define your message types
message HelloRequest {
  string name = 1;
  int32 age = 2;
}

message HelloResponse {
  string message = 1;
  int64 timestamp = 2;
}